.github/workflows/housekeeping-lock-threads.yml

```diff
@@ -0,0 +1,44 @@
+name: "Housekeeping / Lock Threads"
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+
+concurrency:
+  group: lock-threads
+  cancel-in-progress: true
+
+permissions:
+  issues: write  
+  discussions: write
+
+jobs:
+  # example from Pull request / Validation / Copyright & Licenses
+  generate_token:
+    if: ${{ !contains('["Jeredian","Elkin-Vasily","GudwinTheGreat"]', github.actor) }}
+    name: "Validate"
+    runs-on: ubuntu-latest
+    steps:
+    - name: Generate token
+      uses: tibdex/github-app-token@v2.1.0
+      id: generate_token
+      with:
+        app_id: ${{ secrets.APP_ID }}
+        private_key: ${{ secrets.APP_PRIVATE_KEY }}
+
+  change_permissions:
+    name: "permissions"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Lock Outdated Threads"
+        uses: "dessant/lock-threads@v5.0.1"
+        with:
+          github-token: ${{ steps.generate_token.outputs.token }}
+          process-only: 'issues, prs'
+          issue-inactive-days: 14
+          add-issue-labels: "status: outdated"
+          issue-comment: "This issue is closed for editing because the last activity was more than 14 days ago."
```

Comment:
Lock conversation doesn't lock "editing" functionality and issue/pr already closed at that moment. Please set correct comment for issue and pull request. See from example:
```
          issue-comment: >
            This issue has been automatically locked since there
            has not been any recent activity after it was closed.
            Please open a new issue for related bugs.
          pr-comment: >
            This pull request has been automatically locked since there
            has not been any recent activity after it was closed.
            Please open a new issue for related bugs.
```